src/modules/integrator-guide/pages/automation/workflows/examples/messagebus-read-write.adoc

```diff
@@ -0,0 +1,77 @@
+= Messagebus Queue
+:attachment-path: ../../../_attachments/automation/workflows/
+:page-noindex: true
+
+Messagebus queue mechanism complements the workflows in a way that any asynchronous computation can be offloaded to another workflow or workflow-extendable process. +
+This example will be using the xref:automation/workflows/index.adoc[], Integration Gateway xref:api-gw/profiler.adoc[] and the xref:developer-guide:corteza-server/messagebus.adoc[] subsystem.
```

Comment:
Reword to something along the lines of "The workflow used in this example can be found here and the integration gateway can be found here."